pkg/asset/quota/aws/aws.go

```diff
@@ -137,7 +138,12 @@ func machineTypeToQuota(t string, instanceTypes map[string]InstanceTypeInfo) quo
 	switch class {
 	case "a", "c", "d", "h", "i", "m", "r", "t", "z":
 		return quota.Constraint{Name: "ec2/L-1216C47A", Count: info.vCPU}
+	case "g":
+		return quota.Constraint{Name: "ec2/L-3819A6DF", Count: info.vCPU}
 	default:
+		if strings.HasPrefix(strings.ToLower(t), "vt") {
+			return quota.Constraint{Name: "ec2/L-3819A6DF", Count: info.vCPU}
```

Comment:
This is the wrong code. We want on-demand instances not spot instances.
```
$ aws service-quotas list-service-quotas --service-code ec2 --output json | jq '.Quotas[] | select(.QuotaName | contains("On-Demand G and VT instances")) | .QuotaCode' -r
L-DB2E81BA
```

```suggestion
		return quota.Constraint{Name: "ec2/L-DB2E81BA", Count: info.vCPU}
	default:
		if strings.HasPrefix(strings.ToLower(t), "vt") {
			return quota.Constraint{Name: "ec2/L-DB2E81BA", Count: info.vCPU}
```